apps/aens/src/aens_transfer_tx.erl

```diff
@@ -180,7 +180,10 @@ deserialize(?NAME_TRANSFER_TX_VSN,
             , {ttl, TTL}]) ->
     account = aec_id:specialize_type(AccountId),
     name    = aec_id:specialize_type(NameId),
-    account = aec_id:specialize_type(RecipientId),
+    case aec_id:specialize_type(RecipientId) of
+        account -> ok;
+        name -> ok
```

Comment:
Ok, so we will be able to transfer a name to another account, which will be referenced by an address or a name, correct?

Reply:
Correct.

If name, the `account_pubkey` name pointer key will be looked up, and the id type ignored and considered an account.